circuits/src/lib.rs

```diff
@@ -7,57 +7,23 @@
 #![feature(generic_const_exprs)]
 #![feature(inherent_associated_types)]
 
-use crate::zk_circuits::{
-    valid_commitments::SizedValidCommitments, valid_match_mpc::ValidMatchMpcCircuit,
-    valid_reblind::SizedValidReblind, valid_settle::SizedValidSettle,
-    valid_wallet_create::SizedValidWalletCreate, valid_wallet_update::SizedValidWalletUpdate,
-};
-use circuit_types::{
-    errors::{ProverError, VerifierError},
-    traits::{
-        CircuitBaseType, MultiProverCircuit, MultiproverCircuitBaseType,
-        MultiproverCircuitCommitmentType, SingleProverCircuit,
-    },
-};
-use lazy_static::lazy_static;
-use merlin::HashChainTranscript as Transcript;
-use mpc_bulletproof::{
-    r1cs::{Prover, R1CSProof, Verifier},
-    r1cs_mpc::{MpcProver, PartiallySharedR1CSProof},
-    BulletproofGens, PedersenGens,
-};
+use constants::Scalar;
 
 pub mod mpc_circuits;
 pub mod mpc_gadgets;
-mod tracing;
 pub mod zk_circuits;
 pub mod zk_gadgets;
 
-/// The highest possible set bit in the Dalek scalar field
-pub(crate) const SCALAR_MAX_BITS: usize = 253;
-/// The seed for a fiat-shamir transcript
-pub(crate) const TRANSCRIPT_SEED: &str = "merlin seed";
-/// The maximum bit index that may be set in a positive `Scalar`
-pub(crate) const POSITIVE_SCALAR_MAX_BITS: usize = 250;
+// -------------
+// | Constants |
+// -------------
```

Comment:
Many of the helpers/constants in this file are accumulated from the refactor already, they aren't used in this PR, but are used implicitly elsewhere in already refactored code.